integrated_channels/integrated_channel/learner_data.py

```diff
@@ -25,26 +25,30 @@ class BaseLearnerExporter(object):
     Base class for exporting learner completion data to integrated channels.
     """
 
+    GRADE_PASSING = 'Pass'
```

Comment:
I suppose that because SAP is free form, what we are sending them is effectively our default - which hearkens back to what we imagined we would send in a csv to enterprise customers as v1 of data sharing. I think this will be simple enough to change when the next channel that requires specific values comes along.